custom/enikshay/integrations/nikshay/tests/test_repeaters.py

```diff
@@ -88,6 +90,9 @@ def _create_nikshay_registered_case(self):
         )
         self.create_case(nikshay_registered_case)
 
+    def update_case_with(self, case_id, case_attrs):
```

Comment:
elsewhere this is called `update_case`. Would be good to maintain that convention.